main/networking/Runtime.ts

```diff
@@ -120,10 +254,10 @@ class TCPConn {
       this.socket.write(new Uint8Array([1])); // Runtime needs first byte to be 1 to recognize client as Dawn (instead of Shepherd)
     });
 
-    this.socket.on('timeout', () => {
-      this.logger.log('TCP socket timeout');
-      this.socket.end();
-    });
+    // this.socket.on('timeout', () => {
+    //   this.logger.log('TCP socket timeout');
+    //   this.socket.end();
```

Comment:
yea I was testing some stuff and I forgot to remove this and the earlier setTimeout. This might be why Hailey was having some connection issues earlier

Reply:
Yea I remember we did have this timeout stuff before but looking at the documentation again, it says `Sets the socket to timeout after timeout milliseconds of inactivity on the socket`. I guess we missed the `inactivity` part.